src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -514,14 +514,42 @@ export async function getFeatureIdType(output: Log, env: NodeJS.ProcessEnv, user
 	}
 }
 
+export function getBackwardCompatibleFeatureId(id: string) {
+	const migratedfeatures = ['aws-cli', 'azure-cli', 'desktop-lite', 'docker-in-docker', 'docker-from-docker', 'dotnet', 'git', 'git-lfs', 'github-cli', 'java', 'kubectl-helm-minikube', 'node', 'powershell', 'python', 'ruby', 'rust', 'sshd', 'terraform'];
```

Comment:
I see fish, gradle, homebrew, jupyterlab, maven are missing, assuming these are deprecated, should we add a log message when one of these is used?

Reply:
Sounds good. Added a message. 
Once features repo is open source, I'm thinking of adding more detailed log messages for these as well as migrated/renamed features

Reply:
I removed the warning log messages, as per today's discussion. 

I'll add them back only when auto-mapping is ready to be put in Remote-Containers STABLE version

@chrmarti  does that sound correct to you?

Reply:
Makes sense, thanks!